pkg/deploy/defaults.go

```diff
@@ -71,12 +71,28 @@ const (
 	OldDefaultPostgresUpstreamImageToDetect = "centos/postgresql-96-centos7:9.6"
 
 	// ConsoleLink default
-	DefaultConsoleLinkName        = "che"
-	DefaultConsoleLinkImage       = "/dashboard/assets/branding/loader.svg"
-	DefaultConsoleLinkDisplayName = "Eclipse Che"
-	DefaultConsoleLinkSection     = "Red Hat Applications"
+	DefaultConsoleLinkSection              = "Red Hat Applications"
+	DefaultConsoleLinkImage                = "/dashboard/assets/branding/loader.svg"
```

Comment:
is this the Che image? or the CRW image? I'm guessing if we've built the dashboard correctly in CRW it'll use the CRW one, but just wanted to ask.

Reply:
@nickboldt variables names already contain CodeReady, so I just follow the standard that is in place e.g. `defaultCodeReadyServerImageRepo`  https://github.com/eclipse/che-operator/blob/master/pkg/deploy/defaults.go#L24 or `defaultCodeReadyServerImageTag` - https://github.com/eclipse/che-operator/blob/master/pkg/deploy/defaults.go#L24

and I do not plan to do the refactoring. The rule of thumb is either to follow the convention that is already in place (even if you don not like) or proceed with the major refactoring

Reply:
> is this the Che image? or the CRW image?

I suppose it should use CRW image but I can not confirm since I can not verify

Reply:
@nickboldt doesn't the CRW assembly replace the image with a CRW one during packaging ? And in this case is the *relative path* of the image still the same ?
If it's the case we wouldn't need to change the loader image.